### ISSUE TITLE
Numpy FutureWarning

### ISSUE THREAD

**Issue by watsonjj** [fact-project/pycorsikaio#6]
With the latest numpy version (1.17.2) I am getting the following FutureWarning:
```
/Users/Jason/opt/anaconda3/envs/cta/lib/python3.7/site-packages/corsikaio-0.2.1-py3.7.egg/corsikaio/subblocks/dtypes.py:20: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  return np.dtype(dict(**dt))
```

**Comment by MaxNoe** [comment]
@watsonjj thx for reporting, fix in #7

**Comment by MaxNoe** [comment]
New release v0.2.2 is on PyPI
